Description: Don't map 'utf8', 'utf-8' to 'utf', which is not a known encoding for glibc.
Author: Matthias Klose <doko@debian.org>
Origin: Debian cpython packaging
Bug-cpython: http://bugs.python.org/issue1166957
Last-Update: 2011-12-19

--- a/lib-python/2.7/locale.py
+++ b/lib-python/2.7/locale.py
@@ -1592,8 +1592,8 @@
     'uk_ua.iso88595':                       'uk_UA.ISO8859-5',
     'uk_ua.koi8u':                          'uk_UA.KOI8-U',
     'uk_ua.microsoftcp1251':                'uk_UA.CP1251',
-    'univ':                                 'en_US.utf',
-    'universal':                            'en_US.utf',
+    'univ':                                 'en_US.UTF-8',
+    'universal':                            'en_US.UTF-8',
     'universal.utf8@ucs4':                  'en_US.UTF-8',
     'ur':                                   'ur_PK.CP1256',
     'ur_in':                                'ur_IN.UTF-8',
